Phan2.js

```diff
@@ -0,0 +1,79 @@
+//Hàm bt1
+function sum(array) {
+  let sum = 0;
+  if (Array.isArray(array) === false) {
+    console.log("Dữ liệu nhập vào phải là mảng.");
+  } else {
+    for (i = 0; i < array.length; i++) {
```

Comment:
Chỗ này sao không dùng for ...of
Tập dùng for ...of cho quen ngoài việc dùng for bình thường nhé